internal/ghcmd/cmd.go

```diff
@@ -25,11 +25,14 @@ import (
 	"github.com/cli/cli/v2/pkg/iostreams"
 	"github.com/cli/cli/v2/utils"
 	"github.com/cli/safeexec"
-	"github.com/mattn/go-isatty"
 	"github.com/mgutz/ansi"
 	"github.com/spf13/cobra"
 )
 
+// updaterEnabled is a statically linked build property set in gh formula within homebrew/homebrew-core
+// used to control whether users are notified of newer GitHub CLI releases.
+// Development builds leave this empty as impossible to determine if newer or not.
+// For more information, <https://github.com/Homebrew/homebrew-core/blob/master/Formula/g/gh.rb>.
 var updaterEnabled = ""
```

Comment:
Update the comment here calling out that this has to be set to `cli/cli` in order to check the right repo for newer releases.